src/main/java/ch/wisv/events/sales/controller/stats/SalesStatsProductController.java

```diff
@@ -0,0 +1,60 @@
+package ch.wisv.events.sales.controller.stats;
+
+import ch.wisv.events.core.exception.normal.EventNotFoundException;
+import ch.wisv.events.core.model.event.Event;
+import ch.wisv.events.core.service.event.EventService;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+/**
+ * SalesScanEventController.
+ */
+@Controller
+@RequestMapping(value = "/sales/stats/products/{key}")
```

Comment:
If you're going to define only one endpoint per controller, you can also just merge them and define the mapping per method. You can still define a "parent" endpoint `/sales/stats` for all methods in the combined controller.

See https://github.com/AreaFiftyLAN/lancie-api/blob/master/src/main/java/ch/wisv/areafiftylan/users/controller/UserRestController.java#L46 for an example

Reply:
Done